Classes/Controller/GlossaryController.php

```diff
@@ -111,6 +112,15 @@ public function createAction(): void
         }
         $aggregateIdentifier = Algorithms::generateUUID();
 
+
+        foreach ($texts as $language => $text) {
+            if (!empty($this->glossaryEntryRepository->findByTextAndLanguage($text, $language))) {
+                throw new InvalidArgumentException(sprintf('Text "%s" already exists with language "%s".', $text, $language));
+            }
+        }
+
+        $this->addErrorFlashMessage();
```

Comment:
So the Exception above gets automatically transformed to a flash message output?

Reply:
No, unfortunatly not, thats just a leftover from testing. I removed it 👍🏻